### PR TITLE
deal with warning message

### PR DESCRIPTION
**Pull request by sleongmgi**
Deal with the warning message.  Make the debugging message to the standard output to use the diag.

### CODE REVIEW

File: lib/perl/Genome/Sys/Lock/FileBackend.t [hunk 1]
```diff
@@ -24,6 +24,11 @@ use Socket qw(AF_UNIX SOCK_STREAM PF_UNSPEC);
 use Test::More;
 use Time::HiRes qw(gettimeofday);
 
+use Sub::Override;
+
+my $sub_override = Sub::Override->new;
+$sub_override->replace('Genome::Logger::warning' => sub { diag($_[1]); }, 'Genome::Logger::info' => sub { diag($_[1]); });
+
```
Comment (tmooney) [inline lib/perl/Genome/Sys/Lock/FileBackend.t]: Alternatively, we could turn off printing logs to the screen:
```perl
my $logger = Genome::Logger->get();
$logger->screen(0);
```

Comment (sleongmgi) [inline lib/perl/Genome/Sys/Lock/FileBackend.t]: Yes.  I think it might be better to keep the original output.

Comment (tmooney) [inline lib/perl/Genome/Sys/Lock/FileBackend.t]: Fair enough!

**Comment by tmooney** [comment]
+1 👌

**Comment by brummett** [comment]
+1

**Pull request merged**